### PR TITLE
Issue #437: TabsAdapterTest: Remove unnecessary holder3.

### PR DESCRIPTION
**Pull request by Pheonix73**
I removed the line that creates holder3 and kept the call verifyNoMoreInteractions() back to how it was.

### CODE REVIEW

File: components/browser/tabstray/src/test/java/mozilla/components/browser/tabstray/TabsAdapterTest.kt [hunk 1]
```diff
@@ -51,7 +51,7 @@ class TabsAdapterTest {
 
         val holder1 = adapter.createViewHolder(view, 0)
         val holder2 = adapter.createViewHolder(view, 0)
-        val holder3 = adapter.createViewHolder(view, 0)
+
```
Comment (sputnikci) [inline components/browser/tabstray/src/test/java/mozilla/components/browser/tabstray/TabsAdapterTest.kt]: \[ktlint\] WARNING: \[no\-trailing\-spaces\] Trailing space\(s\) in column 1

**Comment by pocmo** [comment]
There are two issues reported by ktlint that prevent us from merging this PR:
> /build/android-components/components/browser/tabstray/src/test/java/mozilla/components/browser/tabstray/TabsAdapterTest.kt:55:1: Needless blank line(s) (cannot be auto-corrected)
> /build/android-components/components/browser/tabstray/src/test/java/mozilla/components/browser/tabstray/TabsAdapterTest.kt:54:1: Trailing space(s) (cannot be auto-corrected)

@Pheonix73 Can you fix that? :)

**Comment by Pheonix73** [comment]
@pocmo  I will try to remove the empty spaces

**Comment by pocmo** [comment]
Thank you @Pheonix73! I squashed, rebased and landed this patch as 527e98abc15011917ce49a43a4f2cd841ee436bf

**Pull request closed without merging**